discovery_engine/lib/src/domain/engine/engine.dart

```diff
@@ -88,56 +58,31 @@ abstract class Engine {
   Future<Set<Source>> getTrustedSources();
 
   /// Adds an excluded source.
-  Future<void> addExcludedSource(
-    Source excluded,
-  );
+  Future<void> addExcludedSource(Source excluded);
 
   /// Removes an excluded source.
-  Future<void> removeExcludedSource(
-    Source excluded,
-  );
+  Future<void> removeExcludedSource(Source excluded);
 
   /// Adds a trusted source.
-  Future<void> addTrustedSource(
-    Source trusted,
-  );
+  Future<void> addTrustedSource(Source trusted);
 
   /// Removes a trusted source.
-  Future<void> removeTrustedSource(
-    Source trusted,
-  );
+  Future<void> removeTrustedSource(Source trusted);
 
   /// Gets the next batch of feed documents.
   Future<List<DocumentWithActiveData>> feedNextBatch();
 
-  /// Gets the next batch of feed documents.
-  Future<List<DocumentWithActiveData>> getFeedDocuments(
-    List<HistoricDocument> history,
-    List<SourceReacted> sources,
-  );
-
-  /// Restores the feed documents, ordered by their global rank (timestamp & local rank).
-  Future<List<DocumentWithActiveData>> restoreFeed();
+  /// Restores the documents which have been fed, i.e. the current feed.
+  Future<List<DocumentWithActiveData>> fed();
```

Comment:
I think the previous name was more descriptive, `engine.fed()` is a bit vague, why you've changed it?

Reply:
mainly to match the `searched` and related naming style. if the new names are vague, i'll rename them to `restore_...`.

Reply:
yeah, I would add `restore*` in front, it reads better imho